Mesh_3/demo/Mesh_3/Volume_plane_thread.h

```diff
@@ -10,12 +10,13 @@
 #include <QThread>
 #include <vector>
 
+template<typename Word>
 struct Clamp_to_one_zero_range {
-  std::pair<float, float> min_max;
-  float operator()(const float& inVal) {
-    float inValNorm = inVal - min_max.first;
-    float aUpperNorm = min_max.second - min_max.first;
-    float bValNorm = inValNorm / aUpperNorm;
+  std::pair<Word, Word> min_max;
+  Word operator()(const Word& inVal) {
+    Word inValNorm = inVal - min_max.first;
+    Word aUpperNorm = min_max.second - min_max.first;
+    Word bValNorm = inValNorm / aUpperNorm;
```

Comment:
You can find the definition of `CGAL_IMAGE_IO_CASE` [here in `CGAL_ImageIO/include/CGAL/ImageIO.h#L629`][1]. The `Word` type of an image can be anything from a `char`, `unsigned int`, and so on, and even floating point values (`float`/`double`). Do you agree with my proposal to keep `Clamp_to_one_zero_range` templatized by `Word` and use the `static_cast` to `float` only to construct the return value of `operator()`?

[1]: https://github.com/CGAL/cgal/blob/72ff845de9e8f815f36410981398b57f18fbac67/CGAL_ImageIO/include/CGAL/ImageIO.h#L629